tests/fixtures/topics.py

```diff
@@ -23,3 +25,14 @@ def topics_test_data():
     file_path = pathlib.Path("src/store/metadata/stub/content/topics.json")
     with open(file_path, "r") as json_file:
         return json.load(json_file)
+
+
+@pytest.fixture
+def sub_topic_test_data():
```

Comment:
so this is just for checking the structure is right and the structure is the same as for topics so just return that and use the doscstring to explain, something like:

```
def sub_topic_test_data():
     """
     explanation
     """
     return topics_test_data()
```

If pytest moans about fixtures calling other fixtures (it might, unsure) then pull out the logic in `topics_test_data` into a stand along function and have both fixtures call it.